composer.json

```diff
@@ -8,7 +8,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.3",
+        "php": "^7.4",
```

Comment:
why you change php version  here?

Reply:
This happens when I convert to the Repository layer structure, I declare a function with input and return results but the program throws an error and automatically asks for php 7.4